docs/.vitepress/strings/ja.json

```diff
@@ -66,188 +66,188 @@
-                  "text": "An XcodeGen project"
+                  "text": "XcodeGen プロジェクト"
                 },
                 "bazel-project": {
-                  "text": "A Bazel project"
+                  "text": "Bazel プロジェクト"
                 }
               }
             }
           }
         },
         "develop": {
-          "text": "Develop",
+          "text": "開発",
           "items": {
             "projects": {
-              "text": "Projects",
+              "text": "プロジェクト",
               "items": {
                 "manifests": {
-                  "text": "Manifests"
+                  "text": "マニフェスト"
                 },
                 "directory-structure": {
-                  "text": "Directory structure"
+                  "text": "ディレクトリ構成"
                 },
                 "editing": {
-                  "text": "Editing"
+                  "text": "編集方法"
                 },
                 "dependencies": {
-                  "text": "Dependencies"
+                  "text": "依存関係"
                 },
                 "code-sharing": {
-                  "text": "Code sharing"
+                  "text": "コードの共有"
                 },
                 "synthesized-files": {
-                  "text": "Synthesized files"
+                  "text": "自動生成ファイル"
                 },
                 "dynamic-configuration": {
-                  "text": "Dynamic configuration"
+                  "text": "動的コンフィギュレーション"
                 },
                 "templates": {
-                  "text": "Templates"
+                  "text": "テンプレート"
                 },
                 "plugins": {
-                  "text": "Plugins"
+                  "text": "プラグイン"
                 },
                 "hashing": {
-                  "text": "Hashing"
+                  "text": "ハッシュ"
                 },
                 "the-cost-of-convenience": {
-                  "text": "The cost of convenience"
+                  "text": "利便性の代償"
                 },
                 "tma-architecture": {
-                  "text": "Modular architecture"
+                  "text": "モジューラーアーキテクチャ"
                 },
                 "best-practices": {
-                  "text": "Best practices"
+                  "text": "ベストプラクティス"
                 }
               }
             },
             "build": {
-              "text": "Build",
+              "text": "ビルド",
               "items": {
                 "cache": {
-                  "text": "Cache"
+                  "text": "キャッシュ"
                 }
               }
             },
             "test": {
-              "text": "Test",
+              "text": "テスト",
               "items": {
                 "smart-runner": {
-                  "text": "Smart runner"
+                  "text": "スマートランナー"
                 },
                 "flakiness": {
-                  "text": "Flakiness"
+                  "text": "Flaky なテスト"
```

Comment:
i think there is no strategy for white space between english word and japanese one. and some place has no white space. please make a strategy of it